ConcernsCaseWork/ConcernsCaseWork/Services/Nti/INtiModelService.cs

```diff
@@ -6,6 +6,13 @@ namespace ConcernsCaseWork.Services.Nti
 {
 	public interface INtiModelService
 	{
-		public Task<ICollection<NtiModel>> GetNtisForCase(long caseUrn);
+		Task<ICollection<NtiModel>> GetNtisForCase(long caseUrn);
```

Comment:
I've seen some methods that return a task suffixed with 'Async'. Is there a preferred standard for this? I've been adding a suffix if I've seen one is missing.